src/components/preview/preview.style.ts

```diff
@@ -1,14 +1,106 @@
-import styled from "styled-components";
+import styled, { css, keyframes } from "styled-components";
 import { margin } from "styled-system";
 import baseTheme from "../../style/themes/base";
+import { Shapes } from "./preview.component";
 
 const StyledPreview = styled.div`
   ${margin}
 `;
 
+interface StyledPreviewPlaceholderProps {
+  height?: string;
+  width?: string;
+  shape: Shapes;
+  isStatic?: boolean;
+  isLastLine: boolean;
+}
+
+const shimmer = keyframes`
+  0% { 
+    opacity: 0.1 
+  }
+  70% { 
+    opacity: 1
+  }
+  100% { 
+    opacity: 0.1 
+  }
+`;
+
+function getBorderRadius(shape: Shapes) {
+  switch (shape) {
+    case "rectangle-round":
+      return "--borderRadius400";
+    case "circle":
+      return "--borderRadiusCircle";
+    default:
+      return "--borderRadius100";
+  }
+}
+
+function getHeight(shape: Shapes) {
+  switch (shape) {
+    case "rectangle":
+      return "40px";
```

Comment:
suggestion: might be worth using the `--sizing` tokens for these values

![image](https://github.com/user-attachments/assets/f877ccfb-db09-4cc9-9c74-7ecf9bf591e0)